ansible/roles/web/templates/parameters.yml.j2

```diff
@@ -45,3 +45,6 @@ parameters:
     # Register at https://sentry.io
     sentry_dsn:        '{{ site_sentry_dsn }}'
     sentry_env:        '{{ site_sentry_env }}'
+
+    env(ALGOLIA_APP_ID):  '{{ site_algolia_app_id }}'
+    env(ALGOLIA_API_KEY): '{{ site_algolia_api_key }}'
```

Comment:
Please add a comment what these parameters are for and where to get them from, and end the file with the newline. (like all files, i.e. fix this across this PR).